core/trino-main/src/main/java/io/trino/operator/HashGenerator.java

```diff
@@ -24,6 +24,13 @@ default int getPartition(int partitionCount, int position, Page page)
         long rawHash = hashPosition(position, page);
         // This function reduces the 64 bit rawHash to [0, partitionCount) uniformly. It first reduces the rawHash to 32 bit
         // integer x then normalize it to x / 2^32 * partitionCount to reduce the range of x from [0, 2^32) to [0, partitionCount)
```

Comment:
This comment can be deleted. It's now a detail of the `partitionUniformly` method